Hudl.Mjolnir/Command/Command.cs

```diff
@@ -38,6 +38,11 @@ public abstract class Command
         protected static readonly ConfigurableValue<bool> UseCircuitBreakers = new ConfigurableValue<bool>("mjolnir.useCircuitBreakers", true);
 
         /// <summary>
+        /// If this is set to true then all calls wrapped in a Mjonir command will ignore the default timeout.
+        /// </summary>
+        protected static readonly ConfigurableValue<bool> IgnoreCommandTimeouts = new ConfigurableValue<bool>("mjolnir.command.ignoreTimeouts",false);
```

Comment:
I'd go with `mjolnir.ignoreTimeouts`, without the middle "command" - configs that start with `mjolnir.command` are usually specific to individual commands, e.g. `mjolnir.command.core-client.GetUser.Timeout=5000`.